services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -360,3 +380,44 @@ pushFeatureConfigEvent tid event = do
   for_
     (newPush (memList ^. teamMemberListType) Nothing (FeatureConfigEvent event) recipients)
     push1
+
+-- | (Currently, we only have 'Public.TeamFeatureConferenceCalling' here, but we may have to
+-- extend this in the future.)
+getFeatureConfigViaAccount :: flag ~ 'Public.TeamFeatureConferenceCalling => UserId -> Galley (Public.TeamFeatureStatus flag)
+getFeatureConfigViaAccount uid = do
+  mgr <- asks (^. manager)
+  brigep <- asks (^. brig)
+  getAccountFeatureConfigClient brigep mgr uid >>= handleResp
+  where
+    handleResp ::
+      Either Client.ClientError Public.TeamFeatureStatusNoConfig ->
+      Galley Public.TeamFeatureStatusNoConfig
+    handleResp (Right cfg) = pure cfg
+    handleResp (Left errmsg) = throwM . internalErrorWithDescription . cs . show $ errmsg
+
+    getAccountFeatureConfigClient ::
+      (HasCallStack, MonadIO m, MonadHttp m) =>
+      Endpoint ->
+      Manager ->
+      UserId ->
+      m (Either Client.ClientError Public.TeamFeatureStatusNoConfig)
+    getAccountFeatureConfigClient brigep mgr = runHereClientM brigep mgr . getAccountFeatureConfigClientM
+
+    getAccountFeatureConfigClientM ::
+      UserId -> Client.ClientM Public.TeamFeatureStatusNoConfig
+    ( _
+        :<|> getAccountFeatureConfigClientM
+        :<|> _
+        :<|> _
+      ) = Client.client (Proxy @IAPI.API)
```

Comment:
Ah, this is pretty ugly. It took me a while to parse it. Wouldn't it make sense to change the API definition to be a record like the other ones, so we wouldn't need to do this?

Reply:
I though about that, and was too lazy.  But you're right, I'll try to make the time!

Reply:
https://github.com/wireapp/wire-server/pull/1818/commits/01955d86cdab6d480c9aa041380bcc6eda54ac0a

i was being a bit lazy.  hope this is better?

Reply:
Sure, it's fine, but the other API definitions are now using the Generic-based approach where you can just define a single record data type that contains all of the endpoints, instead of separate types joined by `:<|>`. See for example https://github.com/wireapp/wire-server/blob/develop/libs/wire-api/src/Wire/API/Routes/Public/Galley.hs#L75.